python/setup.py

```diff
@@ -143,6 +143,7 @@ def find_version(*filepath):
     "click",
     "colorama",
     "pytest",
+    "gputil",
```

Comment:
I think maybe we don't want ray as a whole to depend on this - it'd be good enough to have a soft import (and raise a warning message to install if not found)